### DISCUSSION TITLE
MFA dont work after password change

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#9080] [Unsupported Versions]
When a new user changes the password, then logs off and on again for the new QR code and setup with that code, it fails and says invalid MFA code.

**Comment by dougburks** [comment]
Sounds like what you're describing matches the `IMPORTANT` warning on the MFA page:
![image](https://user-images.githubusercontent.com/1659467/200561797-7e23ed3f-c190-4f91-8029-aa0d1c374253.png)

**Reply by OzzyKampha** [reply]
I tried to follow the message, but stil same problem

**Reply by dougburks** [reply]
Are you using Google Authenticator?

Have you verified that the date and time are correct on the SOC server and the MFA device?

Have you tried resetting the user's password again?

Have you tried creating a new user account?

**Reply by OzzyKampha** [reply]
What is the best way to change the time on the SOC server?

**Reply by dougburks** [reply]
We recommend using NTP to keep the time updated:
https://docs.securityonion.net/en/2.3/ntp.html

**Reply by OzzyKampha** [reply]
I have now check the time. Im able to activate mfa when im connecting direct to the host . But when i connect with the reverse proxy, i get "Unable to save settings: The provided authentication code is invalid, please try again"

**Reply by dougburks** [reply]
Seems like an issue with your reverse proxy or its configuration.
